pkg/secrethub/example_test.go

```diff
@@ -0,0 +1,121 @@
+package secrethub_test
+
+import (
+	"fmt"
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+	"log"
+
+	"github.com/secrethub/secrethub-go/pkg/secrethub/credentials"
+
+	"github.com/secrethub/secrethub-go/pkg/secrethub/iterator"
+)
+
+var client secrethub.ClientInterface
+
+// Create a new Client.
+func ExampleNewClient() {
+	client, err := secrethub.NewClient()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// use the client
+	_, err = client.Repos().Create("workspace/repo")
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Do you have a better suggestion of how to use the client in the example in which we create it?

Reply:
Maybe a `client.Secrets().ReadString`?